server/routes/users.js

```diff
@@ -14,4 +13,41 @@ router.get('/', async function (req, res, next) {
   res.send(allMyUsers);
 });
 
-module.exports = router;
\ No newline at end of file
+
+async function getUserByEmail(email, password) {
+
+  let result = await prisma.user.findFirst({
+    where: {
+      email: email,
+      password: password
+    }
+
+  })
+
+  return result;
+}
+
+
+router.post('/login', async function (req, res, next) {
+  if(!req.body.email || !req.body.password) {
+    res.status(401);
+    res.send("Invalid arguments");
+    return;
+  }
+  const user = await getUserByEmail(req.body.email, req.body.password);
+  if (user == null) {
+
+    res.status(401);
+    res.send("User does not exist!");
+
+  }
+
+  else {
+    const token = await tokenify(user);
+
+    res.send({ token: token });
+  }
+})
+
+module.exports = router;
+module.exports.getUserByEmail=getUserByEmail
```

Comment:
This function is defined in `/server/persist/users.js` now, right? Can we remove it from here? this is basically the same as saying `router.getUserByEmail=getUserByEmail` which feels wrong to me

Reply:
actually I think all the changes in this file can get reverted, this looks like some git branch merging black-magic resulting in some changes getting undone